.rubocop_todo.yml

```diff
@@ -53,7 +53,7 @@ Metrics/AbcSize:
 # Offense count: 7
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 406
+  Max: 410
```

Comment:
this got longer as the ItemsContoller got longer so I could fix the redirect issue with the bulk action call